cdc/redo/manager.go

```diff
@@ -532,27 +566,26 @@ func (m *ManagerImpl) onResolvedTsMsg(tableID model.TableID, resolvedTs model.Ts
 
 func (m *ManagerImpl) bgUpdateLog(ctx context.Context, errCh chan<- error) {
 	logErrCh := make(chan error, 1)
-	handleErr := func(err error) {
-		select {
-		case logErrCh <- err:
-		default:
-		}
-	}
+	handleErr := func(err error) { logErrCh <- err }
```

Comment:
please add a comment about your consideration here, why a buffered channel for error ?